### PR TITLE
GreenEye Monitor component docs

### PR DESCRIPTION
**Pull request by jkeljo**
**Description:**
Adding documentation for the new GreenEye Monitor component.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#16378

## Checklist:

- [X] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [X] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Review by DubhAd — DISMISSED** [review]
👍

File: source/_components/greeneye_monitor.markdown [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+      type: positive_int
+    channels:
+      description: The list of channels that should appear in Home Assistant for this monitor. Data from other channels will be ignored.
+      required: false
+      type: list
+      keys:
+        number:
+          description: The channel number as it appears in the GEM UI.
+          required: true
+          type: positive_int
```
Comment (frenck) [inline source/_components/greeneye_monitor.markdown]: `postive_int` is not a valid documentation type.

```suggestion
          type: integer
```

File: source/_components/greeneye_monitor.markdown [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+  type: string
+monitors:
+  description: The list of monitors that should appear in Home Assistant. Data from other monitors will be ignored.
+  required: false
+  type: list
+  keys:
+    serial_number:
+      description: The last five digits (omitting leading zeroes) of your GEM serial number.
+      required: true
+      type: positive_int
```
Comment (frenck) [inline source/_components/greeneye_monitor.markdown]: `postive_int` is not a valid documentation type.
```suggestion
      type: integer
```

File: source/_components/greeneye_monitor.markdown [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+          required: false
+          type: bool
+          default: false
+    temperature_sensors:
+      description: Configuration for temperature sensors
+      required: false
+      keys:
+        temperature_unit:
+          description: The unit of measure to use for the temperature (F or C)
+          type: temperature_unit
```
Comment (frenck) [inline source/_components/greeneye_monitor.markdown]: `temperature_unit` is not a valid documentation type.

```suggestion
          type: string
```

File: source/_components/greeneye_monitor.markdown [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+          required: true
+        sensors:
+          description: The list of temperature sensors that should appear in Home Assistant for this monitor. Data from other sensors will be ignored.
+          required: true
+          type: list
+          keys:
+            number:
+              description: The sensor number as it appears in the GEM UI.
+              required: true
+              type: positive_int
```
Comment (frenck) [inline source/_components/greeneye_monitor.markdown]: `postive_int` is not a valid documentation type.
```suggestion
              type: integer
```

File: source/_components/greeneye_monitor.markdown [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+              type: string
+    pulse_counters:
+      description: The list of pulse counters that should appear in Home Assistant for this monitor. Data from other pulse counters will be ignored.
+      required: false
+      type: list
+      keys:
+        number:
+          description: The pulse counter's number as it appears in the GEM UI.
+          required: true
+          type: positive_int
```
Comment (frenck) [inline source/_components/greeneye_monitor.markdown]: `postive_int` is not a valid documentation type.
```suggestion
          type: integer
```

File: source/_components/greeneye_monitor.markdown [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+          type: string
+        counted_quantity_per_pulse:
+          description: The number of the counted quantity represented by each pulse.
+          required: false
+          type: float
+          default: 1.0
+        time_unit:
+          description: The time unit to use when computing rates (s, min, or h)
+          required: false
+          type: s, min, h
```
Comment (frenck) [inline source/_components/greeneye_monitor.markdown]: `s, min, h` is not a valid documentation type.

```suggestion
          type: string
```

File: source/_components/greeneye_monitor.markdown [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+description: "Instructions on how to integrate your GreenEye Monitor within Home Assistant."
+date: 2018-09-02 11:15
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: brultech.png
+ha_category: Hub
+featured: false
+ha_release: 0.75
```
Comment (frenck) [inline source/_components/greeneye_monitor.markdown]: `0.75` has been shipped already, please update to `0.82` (nice starting point)

File: source/_components/sensor.greeneye_monitor.markdown [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+description: "Instructions on how to integrate the various sensors on your Greeneye Monitor into Home Assistant."
+date: 2018-09-02 14:13
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: brultech.png
+ha_category: Sensor
+ha_iot_class: "Local Push"
+ha_release: 0.75
```
Comment (frenck) [inline source/_components/sensor.greeneye_monitor.markdown]: `0.75` has been shipped already, please update to `0.82` (nice starting point)

File: source/_components/greeneye_monitor.markdown [hunk 9]
```diff
@@ -0,0 +1,127 @@
...
+        number:
+          description: The pulse counter's number as it appears in the GEM UI.
+          required: true
+          type: positive_int
+        name:
+          description: The name that should be used for this pulse counter in Home Assistant.
+          required: true
+          type: string
+        counted_quantity:
+          description: The unit being counted by this pulse counter (e.g. gal, L)
```
Comment (frenck) [inline source/_components/greeneye_monitor.markdown]:
```suggestion
          description: The unit being counted by this pulse counter (e.g., gal, L)
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
Thanks for the PR @jkeljo, I've reviewed it, and left some comments.
Could you please take a look? 👍

File: source/_components/greeneye_monitor.markdown [hunk 10]
```diff
@@ -0,0 +1,127 @@
...
+title: "GreenEye Monitor"
+description: "Instructions on how to integrate your GreenEye Monitor within Home Assistant."
+date: 2018-09-02 11:15
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: brultech.png
+ha_category: Hub
+featured: false
```
Comment (MartinHjelmare) [inline source/_components/greeneye_monitor.markdown]: This is the default.

**Review by frenck — APPROVED** [review]

**Pull request merged**